File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UserPostActivity/UserPostView.java
package com.smedialink.abakarmagomedov.githubapplication.presentation.UserPostActivity;

import com.smedialink.abakarmagomedov.githubapplication.data.entity.Repository;
import com.smedialink.abakarmagomedov.githubapplication.data.entity.User;

import java.util.List;

/**
 * Created by abakarmagomedov on 15/05/17.
 */

public interface UserPostView {

    void fetchData(List<Repository> reposList);
    void showDialog();
    void hideDialog();
    void error();

}

File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UserPostActivity/UserPostView.kt
package com.smedialink.abakarmagomedov.githubapplication.presentation.UserPostActivity

import com.smedialink.abakarmagomedov.githubapplication.data.entity.Repository

/**
 * Created by abakarmagomedov on 15/05/17.
 */
interface UserPostView {
    fun fetchData(reposList: List<Repository>)
    fun showDialog()
    fun hideDialog()
    fun error()
}
